inst/extdata/default.yaml

```diff
@@ -1,5 +1,5 @@
-xaxs: "i"
-yaxs: "i"
+xaxs: "r"
```

Comment:
Let's keep it in so that we can get the bugs on the user specified limits fixed. Obviously our yaml is going to change a lot before we introduce the package to the wild, and picking our favorite defaults might take some back-and-forth discussions.